docs/design/run.md

```diff
@@ -145,12 +147,11 @@ operations created by the subprocess will be ignored.
 
 ### Rewriting the revisions 
 
-We should handle public and private revisions differently. We choose to operate
-on an immutable history by default.
+Like all commands, `jj run` will refuse to rewrite public/immutable commits.
 
 ### Public revisions
 
-For published revisions, we will not allow `jj run?  to modify them and then 
+For published revisions, we will not allow `jj run`  to modify them and then 
```

Comment:
nit: this change belongs in the first commit

Reply:
done.